### ISSUE TITLE
Token?

### ISSUE THREAD

**Issue by XerockXMG** [sbueringer/grafana-consul-datasource#3]
I'm assuming the token it is asking for is the ACL token, correct?

I ran into the same issue as #1 but also managed to fix it. But now I am just getting HTTP Error Internal Server Error no matter which ACL token I try to provide it.

**Comment by sbueringer** [comment]
Yes the token is the usual ACL token which is then used by the plugin to read the data from Consul. There's usually some kind of indication what's wrong in the Consul log if the internal server error is from Consul, if not Grafana probably logged something

**Comment by TaboolaDevOpsHomeAssignment** [comment]
@sbueringer Are you planning by any chance to allow access without token?

**Comment by sbueringer** [comment]
@TaboolaDevOpsHomeAssignment I actually never thought of running Consul without token. But I guess it's possible? Then I think nothing speaks against supporting that.
